assets/meta.json

```diff
@@ -1,11 +1,11 @@
 {
   "title": "The Reactivate London Meetup",
-  "description": "Join us for our fifth meetup with talks ranging from React to this instalment's special topic: Games! Pizza and drinks on the house, of course.",
-  "date": "Aug 15th",
+  "description": "Join us for our sixth meetup with an Elm talk and our first Reactivate quiz! Pizza and drinks on the house, of course.",
```

Comment:
- can we change "pizza and drinks" to "Food and Drinks" 
- Elm talk to Elm talk/crash course

Reply:
Is it something different than pizza this time? I’d love that haha

Also, talk/crash course is a little clunky. Can we settle on one or the other?

Reply:
No, it's probably pizza but I'd like the option